keyboards/atreus/keymaps/ridingqwerty/atreus.h

```diff
@@ -0,0 +1,38 @@
+/* Copyright 2018 Jack Humbert
```

Comment:
You shouldn't need this file in your keymap folder

Reply:
This is a handwire so I figured a keymap specific `LAYOUT` was required -- is there a better place for that at the keymap level?

Reply:
Just did a quick spelunk through the repo.  I see one or two instances of a LAYOUT being in a "keymap.h" file, and quite a few where it's built directly into "config.h" -- would either of these be an acceptable alternative to including "<board>.h"?

Reply:
Moved LAYOUT to keymap.c